wsmaster/che-core-api-machine-shared/src/main/java/org/eclipse/che/api/machine/shared/dto/execagent/event/DtoWithPidDto.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.machine.shared.dto.execagent.event;
+
+/**
+ * @author Dmitry Kuleshov
+ */
+public interface DtoWithPidDto {
```

Comment:
I mean that this dto would be a model interface that describes an abstract process that might have a identifier. And inherited interfaces would be an actual data transfer objects, like it has been done with `org.eclipse.che.api.core.model.project.ProjectConfig` and `org.eclipse.che.api.workspace.shared.dto.ProjectConfigDto`.